etc/wazo-auth-keys/config.yml

```diff
@@ -195,6 +195,12 @@ wazo-webhookd:
     - 'auth.users.*.external.mobile.read'
     - 'auth.users.*.read'
 
+wazo-websocketd:
+  system_user: wazo-websocketd
+  acl:
+    - 'auth.tenants.read'
+    - 'auth.tokens.read'
```

Comment:
Why did you need this ACL? This is to list refresh token

Reply:
Actually not needed, mistakenly copied from example